### PR TITLE
Delete azure cosmos DB sensor example_dag

### PR DESCRIPTION
**Pull request by amoghrajesh**
Azure cosmos DB tests are already present in the systests: https://github.com/apache/airflow/blob/main/tests/system/providers/microsoft/azure/example_azure_cosmosdb.py

We need to remove them as part of the effort to migrating the example DAGS to system tests.
related: #32295 

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by amoghrajesh** [comment]
@Adaverse you can have a look here. This was already migrated earlier.

**Comment by Adaverse** [comment]
This test does not comply with the premise of being self-contained.

Maybe we have a flow like assuming a cosmos db already exists, we can parallelly start two tasks, one the sensor and the other a task that sleeps for few seconds doing nothing, and then after few seconds it creates a document which the sensor can sense and finally deleting the document on the success of all the tasks. Here is an example -https://github.com/apache/airflow/blob/main/tests/system/providers/sftp/example_sftp_sensor.py

WDYT?

**Review by Lee-W — APPROVED** [review]

**Comment by amoghrajesh** [comment]
> This test does not comply with the premise of being self-contained.
> 
> Maybe we have a flow like assuming a cosmos db already exists, we can parallelly start two tasks, one the sensor and the other a task that sleeps for few seconds doing nothing, and then after few seconds it creates a document which the sensor can sense and finally deleting the document on the success of all the tasks. Here is an example -https://github.com/apache/airflow/blob/main/tests/system/providers/sftp/example_sftp_sensor.py
> 
> WDYT?

Yes that makes sense. But the intention of this PR is to delete the example_dag instead. I will handle that in a separate PR later. @Adaverse

**Comment by amoghrajesh** [comment]
@Adaverse @potiuk the intention of this PR is to remove the example dag for azure cosmos DB. Can you have a look when you have some time?

**Review by Adaverse — APPROVED** [review]
As a part of cleanup, looks good!

**Review by phanikumv — COMMENTED** [review]
typo in the file name `docs/apache-airflow-providers-microsoft-azure/sensors/cosmos_document_**sesnor**.rst`

**Comment by amoghrajesh** [comment]
Haha good catch @phanikumv
It was already faulty looks like. Pushing a fix

File: tests/system/providers/microsoft/azure/example_azure_cosmosdb.py [hunk 1]
```diff
@@ -48,12 +48,13 @@
     doc_md=__doc__,
     tags=["example"],
 ) as dag:
-
+    # [START cosmos_document_sensor]
     t1 = AzureCosmosDocumentSensor(
         task_id="check_cosmos_file",
         collection_name="airflow_example_coll",
         document_id="airflow_checkid",
     )
```
Comment (hussein-awala) [inline tests/system/providers/microsoft/azure/example_azure_cosmosdb.py:56]: The `database_name` parameter doesn't have a default value, you should add it.

**Review by hussein-awala — COMMENTED** [review]
Same for `AzureCosmosInsertDocumentOperator`.

Could you fix that in the same PR?

File: tests/system/providers/microsoft/azure/example_azure_cosmosdb.py (hunk 1 above)
Comment (amoghrajesh) [reply tests/system/providers/microsoft/azure/example_azure_cosmosdb.py:56]: It is present at the DAG level - line 45

**Comment by amoghrajesh** [comment]
> Same for `AzureCosmosInsertDocumentOperator`.
> 
> Could you fix that in the same PR?

Didn't get you, what should I do for that operator?

File: tests/system/providers/microsoft/azure/example_azure_cosmosdb.py (hunk 1 above)
Comment (hussein-awala) [reply tests/system/providers/microsoft/azure/example_azure_cosmosdb.py:56]: Sorry, I didn't notice that.

**Review by hussein-awala — APPROVED** [review]

**Pull request merged**